CharactersAndStrings.java

```diff
@@ -0,0 +1,81 @@
+
+public class CharactersAndStrings {
+
+    public static int findSymbolOccurance(String str, char ch){
+        int coint = 0;
+
+        for(int i = 0; i < str.length(); i++){
+            if(str.charAt(i) == ch){
+                coint++;
+            }
+        }
+        return coint;
+    }
+
+    public static int findWordPosition(String source, String target){
+        StringBuilder str = new StringBuilder(source);
+
+        return str.indexOf(target);
+    }
+
+    public static String stringReverse(String str){
+        StringBuilder builder = new StringBuilder(str);
+
+        return builder.reverse().toString();
+    }
+
+    public static boolean isPalindrome(String str){
+
+        return str.equals(CharactersAndStrings.stringReverse(str));
```

Comment:
Это лишнее, ты в одном и тоже классе располагаешься.
```suggestion
        return str.equals(stringReverse(str));
```